src/test/hammerdb/README.md

```diff
@@ -0,0 +1,15 @@
+
+# How to trigger hammerdb benchmark jobs
+
+You can trigger two types of hammerdb benchmark jobs:
+-ch_benchmark (analytical and transactional queries)
+-tpcc_bnehcmark (only transactional queries)
+
+If need to prepend `ch_benchmark/` or `tpcc_benchmark/` prefix to your branch to trigger these jobs
```

Comment:
Can you improve this sentence?

Reply:
I improved the readme in general, your branch will be run against master when these jobs are used. Also included that in the readme.